src/seismometer/configuration/model.py

```diff
@@ -171,6 +172,22 @@ class Event(BaseModel):
     The strategy for aggregating (or selecting) scores for an event.
     Supports min, max, first, and last; defaulting to max.
     """
+    merge_strategy: Optional[MergeStrategies] = "forward"
+    """
+    The strategy for merging events with predictions.
+    Supports first, last, nearest, forward, and count; defaulting to forward.
+
+    | - first: The first event within the window is selected. If no window, the first event is selected.
+    | - last: The last event within the window is selected. If no window, the last event is selected.
+    | - nearest: The event closest to the prediction time within the window is selected.
+                 If no window, the nearest event is selected.
+    | - forward: The first event at or after the prediction time within the window is selected.
+                 If no window, the first event at or after the prediction time is selected.
```

Comment:
Do we need a `backwards` for the most recent event *before* the prediction (basically nearest but before?)

I guess the other question is do we need `nearest` if we have `forward` since the only different is that nearest can look into the past?

Reply:
We actually discussed that a little bit previously [here](https://github.com/epic-open-source/seismometer/pull/76#discussion_r1727774320). Admittedly, the exclusion of "backwards" was a bit of an oversight on my part, but the conclusion seemed to be it could be a separate enhancement if it seems useful. 

I'm not presently aware of a use case for nearest, but it was easy to include while I was in the middle of adding everything and seems plausible there may be a use case at some point.